github/src/main/scala/scabot/github/GithubApi.scala

```diff
@@ -94,8 +96,8 @@ trait GithubApiTypes { self: core.Core =>
                                 // diff_hunk, original_position, original_commit_id
 
   case class PullRequestEvent(action: String, number: Int, pull_request: PullRequest) extends ProjectMessage with PRMessage
-  case class PushEvent(ref: String, before: String, after: String, created: Boolean, deleted: Boolean, forced: Boolean,
-                       base_ref: Option[String], commits: List[CommitInfo], head_commit: CommitInfo, repository: Repository, pusher: Author)
+  case class PushEvent(ref_name: String, distinct_commits: List[CommitInfo], repository: Repository) extends ProjectMessage
+//                        ref: String, before: String, after: String, created: Boolean, deleted: Boolean, forced: Boolean,  base_ref: Option[String], commits: List[CommitInfo], head_commit: CommitInfo, repository: Repository, pusher: Author)
```

Comment:
yeah, i left that in because it's painful to convert the json stuff to case classes -- it's not super pretty, but I'll leave these here